recipe/bld.bat

```diff
@@ -3,7 +3,7 @@ pushd "%SRC_DIR%"\cpp\build
 
 set ARROW_BUILD_TOOLCHAIN=%LIBRARY_PREFIX%
 
-cmake -G "Visual Studio 14 2015 Win64" ^
+cmake -G "Visual Studio %VS_MAJOR% %VS_YEAR% Win64" ^
```

Comment:
`cmake -G "%CMAKE_GENERATOR%" ^`

Reply:
@isuruf ah, thanks. That is even better :)